examples/components/NER/traininsilo/run.py

```diff
@@ -180,13 +180,13 @@ def __init__(
         logger.info(f"Trainable parameters: {trainable_params}")
 
         self.model_.train()
+        self.model_.to(self.device_)
```

Comment:
Is .train() interfering with .to() somehow? I wonder if it's more proper to first do .to() then only .train()

Reply:
Should not matter according to documentation: https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module.train 

The only reason why this was moved here from line 189 is that DDP is conditioned under having model in GPU.

Reply:
If this is not required for DDP, we can move this to line 194 as a model has to be in the `training mode` when applying changes for `differential-privacy`.(By default, huggingface loads a model in the `eval mode`)